features/step_definitions/admin_steps.rb

```diff
@@ -0,0 +1,130 @@
+Given(/^I have an admin account$/) do
+  @admin = AdminUser.create(email: "admin@example.com", password: "password", 
+    password_confirmation: "password")
+  # byebug
+end
+
+When(/^I visit the admin panel url$/) do
+  visit("/admin/login")
+end
+
+When(/^I enter my admin email$/) do
+  fill_in "Email*", with: "admin@example.com"
+end
+
+When(/^I enter my admin password$/) do
+  fill_in "Password*", with: "password"
+end
+
+When(/^I click submit$/) do
+  click_button("Login")
+end
+
+Then(/^I see the admin panel$/) do
+  expect(page).to have_content("Dashboard")
+end
+
+Given(/^I do not have an admin account$/) do
+  AdminUser.delete_all
+end
+
+Then(/^I see a flash notification that tell me that my email does not exist in the system$/) do
+  expect(page).to have_content("Invalid email or password")
+end
+
+Given(/^I am logged into the admin panel$/) do
+  @admin = AdminUser.create(email: "admin@example.com", password: "password", 
+    password_confirmation: "password")
+    visit("/admin")
+    fill_in "Email*", with: "admin@example.com"
+    fill_in "Password*", with: "password"
+    click_button("Login")
+end
+
+When(/^I visit the admin books url$/) do
+  visit("/admin/books")
+end
+
+When(/^I want to add a book$/) do
+  visit("/admin/books/new")
+end
+
+When(/^I enter the title "(.*?)"$/) do |title|
+  fill_in "Title", with: title
+end
+
+When(/^I enter the price "(.*?)"$/) do |price|
+  fill_in "Price cents", with: price
+end
```

Comment:
See if you can DRY up this and the previous `When`, using regex. Let me know if you need some help.